src/transport/primus/server.js

```diff
@@ -38,7 +38,7 @@ PrimusTransport.prototype._connect = function(url,port){
 	var address = url+':'+port;
 	if(!this._connections[address]){
 		this._connections[address] = new this._Studio.promise(function(resolve,reject){
-            var client = new _self._server.Socket(url+":"+port,_self.clientOpt);
+            var client = new _self._server.Socket('ws://' + url+":"+port,_self.clientOpt);
```

Comment:
@avishnyak we cant just add ws:// because some users might want to use wss (ssl) we will need a configuration for this